src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Ball {
+    public String color;
```

Comment:
The 'color' field should be private to ensure proper encapsulation. Consider changing it to 'private String color;'. 🛠️